### ISSUE TITLE
slider: Jaws does not read anything when focus goes on slider

### ISSUE THREAD

**Issue by gauravlodha** [angular/material#11685]
## Bug, enhancement request, or proposal:
Bug

#### Detailed Reproduction Steps: 
1.  go to slider demo https://material.angularjs.org/latest/demo/slider
1. Press tab to move focus on slider

### What is the expected behavior?
Jaws should read arial label content

### What is the current behavior?
Jaws does not reading anything

### Which versions of AngularJS, Material, OS, and browsers are affected?

- AngularJS:  1.7.7
- AngularJS Material: 1.1.13, 1.1.14
- OS:  Windows 10
- Browsers: Chrome

**Comment by Splaktar** [comment]
I don't have JAWS to test with, but using VoiceOver I am seeing a number of sliders in the demo only read `<value>, group` and ignore the `aria-label` while others read it just fine. I'll investigate this a bit more and hopefully you can help me test this on JAWS when I have it fixed on VoiceOver.

**Comment by Splaktar** [comment]
There was a change made in PR https://github.com/angular/material/pull/10731 to fix https://github.com/angular/material/issues/10565 for ChromeVox in `1.1.6` because the `role="slider"` was being placed on the `md-slider-wrapper` instead of the `md-slider. 

This appears to have broken JAWS and VoiceOver as testing on `1.1.5` is much better in VoiceOver. Can you please verify that you are seeing that same with JAWS? You can test here: https://material.angularjs.org/1.1.5/demo/slider

**Comment by Splaktar** [comment]
It looks like this was a bug in ChromeVox that has since been fixed. Reverting the change in PR https://github.com/angular/material/pull/10731, making some tweaks to the demos, and moving some of the `aria` attributes to the `md-slider-wrapper` causes VoiceOver to work properly again and seems to also work fine in ChromeVox.

@gauravlodha can you please test this fix on JAWS [here](https://angularjs-material.firebaseapp.com/demo/slider)?

**Comment by Splaktar** [comment]
I've opened PR https://github.com/angular/material/pull/11688 to resolve this, but I would like to get confirmation that it works properly on JAWS before merging it. Also please indicate what version of JAWS is in use.

**Comment by gauravlodha** [comment]
@Splaktar 
thanks for the looking into the issue and I have checked on the above link given by you and its working fine for me and I have used JAWS 2019.1903.47
